tools/_setup_generation/step_rest_refman.py

```diff
@@ -122,10 +122,16 @@ def get_property_type(property_desc, from_schemas):
                     file.write("\n")
                 request_body = method_desc.get("requestBody")
                 if request_body is not None:
-                    file.write(f'<div><h4 style="display: inline-block;">Request body:</h4>\n')
-                    schema = request_body["content"]["application/json"]["schema"]["$ref"]
+                    file.write(
+                        f'<div><h4 style="display: inline-block;">Request body:</h4>\n'
+                    )
+                    schema = request_body["content"]["application/json"]["schema"][
+                        "$ref"
+                    ]
                     schema = schema[schema.rfind("/") + 1 :]
-                    file.write(f'<span><a href="schemas.md#{schema.lower()}">{schema}</a></div>\n\n')
+                    file.write(
+                        f'<span><a href="schemas.md#{schema.lower()}">{schema}</a></div>\n\n'
+                    )
```

Comment:
I believe your automatic formatting is incorrect. Can you check?

Reply:
I am using Black to format the code. What do I have to do?